app/Components/Navigation/Nav.module.scss

```diff
@@ -0,0 +1,7 @@
+.nav {
```

Comment:
i dont think these styles are enough

Reply:
It worked in the design with different things i tested what do i add